project/cmake/scripts/ios/archsetup.cmake

```diff
@@ -12,10 +12,8 @@ set(CMAKE_SYSTEM_NAME Darwin)
 if(WITH_ARCH)
   set(ARCH ${WITH_ARCH})
 else()
-  if(CPU STREQUAL armv7)
-    set(ARCH arm-apple-darwin)
-  elseif(CPU STREQUAL arm64)
-    set(ARCH arm-apple-darwin)
+  if(CPU STREQUAL armv7 OR CPU STREQUAL arm64)
+    set(ARCH arm-osx)
```

Comment:
this is bullshit from some addon dylibs or so ... at one point i will hopefully clean it up *grrr*